packages/fxa-settings/src/components/Ready/index.stories.tsx

```diff
@@ -49,20 +49,23 @@ export const ResetPasswordConfirmForLoggedOutUser = () => (
 
 export const ResetPasswordConfirmedWithRelyingParty = () => (
   <ReadyWithLayout
+    isSignedIn
```

Comment:
Hmm should we set this on `AppContext` instead of propdrilling?

Reply:
I think since it's just one layer it might be preferable for readability? But open to doing it on the context!

Reply:
This feels like a follow up - as needed? It would for sure make sense to move this into context if we need to check on signed in status for multiple pages (e.g., if a user is already signed in, will we need to redirect them forward from the /signin /signup pages?)